src/hotspot/share/opto/vector.cpp

```diff
@@ -413,15 +414,17 @@ void PhaseVector::expand_vunbox_node(VectorUnboxNode* vec_unbox) {
 
     Node* mem = vec_unbox->mem();
     Node* ctrl = vec_unbox->in(0);
-    Node* vec_field_ld = LoadNode::make(gvn,
-                                        ctrl,
-                                        mem,
-                                        vec_adr,
-                                        vec_adr->bottom_type()->is_ptr(),
-                                        TypeOopPtr::make_from_klass(field->type()->as_klass()),
-                                        T_OBJECT,
-                                        MemNode::unordered);
-    vec_field_ld = gvn.transform(vec_field_ld);
+    Node* vec_field_ld;
+    {
+      DecoratorSet decorators = C2_READ_ACCESS | C2_CONTROL_DEPENDENT_LOAD | IN_HEAP;
```

Comment:
C2_READ_ACCESS will be set by "bs->load_at" so you can skip that. 
MO_UNORDERED is missing. That corresponds to "MemNode::unordered" in the original code.

Reply:
`C2_CONTROL_DEPENDENT_LOAD` is also redundant (though original code does that): it's just a plain load from a final instance field).

Reply:
Fixed in the new pr: https://github.com/openjdk/jdk16/pull/139